.gitignore

```diff
@@ -0,0 +1,26 @@
+.snowpack
+build
+node_modules
+
+.eslintcache
+.vscode
+.idea
+
+.yarn/*
```

Comment:
Please do not ignore cache.

For node modules see https://github.com/yarnpkg/berry/discussions/2621 and https://github.com/yarnpkg/berry/issues/2528

To use it with actions, see https://github.com/actions/cache/blob/main/examples.md#node---yarn-2